rules/windows/discovery_whoami_command_activity.toml

```diff
@@ -17,31 +17,31 @@ false_positives = [
 ]
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Whoami Process Activity"
 risk_score = 21
 rule_id = "ef862985-3f13-4262-a686-5f357bbb9bc2"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and process.name:whoami.exe
+process where event.type in ("start", "process_started") and process.name : "whoami.exe"
```

Comment:
whoami is a bit better, has more potential (could use parent, active user, args, etc)

Reply:
this one is on our list to deprecate as well